CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.0-beta.0] - Unreleased
+### Changed
+- The library is now an Application and handles DB connections.
```

Comment:
This is breaking too since it requires the dependents to add the library in their supervision tree.